### DISCUSSION TITLE
Change license to allow use at Google?

### DISCUSSION THREAD

**Discussion by tom4everitt** [aibasel/pyperplan#24] [General]
Hi,

I'm an AGI safety / alignment researcher at Google DeepMind, and I'd like to use your package for parsing PDDL problems to test language models on. I've played around with it in a personal capacity, and I'm really impressed with it so far.

Unfortunately, I'm not able to use it at work / inside Google, due to the GPL license. So I just wanted to ask if you would consider changing the license to a more permissive one, like Apache or MIT. Of course I fully respect if you've made a deliberate decision to use GPL, but I just thought I'd ask since sometimes people pick a license more or less at random.

Either way, congratulations on creating such a solid package!

Best
Tom Everitt
tomeveritt.se

**Comment by jendrikseipp** [comment]
Hi Tom,

Since Pyperplan has been created jointly by many former students, we can't easily ask for everybody's permission to relicense the project. However, there are other projects that might fit your needs. For example, the Unified Planning library uses the Apache license (https://github.com/aiplan4eu/unified-planning). There is also an upcoming project which we haven't tried ourselves: https://github.com/AI-Planning/pddl/ provides a Python PDDL parser using the MIT license.
